moped-database/migrations/1723496266686_add_bike_share_station_comp/up.sql

```diff
@@ -0,0 +1,37 @@
+-- add bike share station component type (point) to moped_components table if it doesn't already exist
+INSERT INTO moped_components (component_name, line_representation, feature_layer_id)
+SELECT
+    'Bike Share Station',
+    FALSE,
+    5
+WHERE NOT EXISTS (
+        SELECT *
+        FROM
+            moped_components
+        WHERE (component_name, line_representation, feature_layer_id) = ('Bike Share Station', FALSE, 5)
+    );
+
+-- insert new, mod, and replacement work types for bike share station in moped_component_work_types table
+WITH inserts_todo AS (
+    SELECT
+        mwt.id AS work_type_id,
```

Comment:
kinda confused about how this sql is working, how does the code know what table `mwt` or `mc` is? what am i missing :sweat_smile: